pkg/serial/handshake.go

```diff
@@ -162,35 +158,11 @@ func HandshakeClient(ctx context.Context, conn net.Conn, debug bool) error {
 	return nil
 }
 
-func readMultiple(conn net.Conn, b []byte) (int, error) {
-	if runtime.GOOS != "windows" {
-		return conn.Read(b)
-	}
-
-	// we want a blocking read, but the behaviour described in the remarks section
```

Comment:
please don't just remove the comments relating to windows - they'll be useful to have around for when we pick that work up again.